app/src/main/res/values/strings.xml

```diff
@@ -16,6 +16,12 @@
     <string name="hide_key_config">Hide hotkey configurations</string>
     <string name="_not_available_">(Not Available)</string>
     <string name="button_haptic_feedback">Haptic feedback on keypress</string>
+    <string name="button_vibration_press_milliseconds_arguments">Press vibration duration arguments</string>
+    <string name="button_vibration_press_milliseconds">Press vibration duration</string>
+    <string name="button_vibration_long_press_milliseconds">Long press vibration duration</string>
+    <string name="button_vibration_press_amplitude_arguments">Press vibration amplitude arguments</string>
+    <string name="button_vibration_press_amplitude">Press vibration amplitude</string>
+    <string name="button_vibration_long_press_amplitude">Long press vibration amplitude</string>
```

Comment:
I'd suggest to change those strings to:

```suggestion
    <string name="button_vibration_milliseconds">Keypress vibration duration</string>
    <string name="button_press">Press</string>
    <string name="button_long_press">Long press</string>
    <string name="button_vibration_amplitude">Keypress vibration amplitude</string>
```

and use them like:

```kotlin
val (primary, secondary) = twinInt(
    title = R.string.button_vibration_milliseconds,
    label = R.string.button_press,
    key = "button_vibration_press_milliseconds",
    defaultValue = 0,
    secondaryLabel = R.string.button_long_press,
    secondaryKey = "button_vibration_long_press_milliseconds",
    secondaryDefaultValue = 0,
    min = 0,
    max = 100,
    unit = "ms"
)
```

to avoid unnecessary repetition